platforms/nuttx/src/bootloader/microchip/mpfs/main.c

```diff
@@ -917,6 +917,7 @@ bootloader_main(int argc, char *argv[])
 {
 	unsigned timeout = BOOTLOADER_DELAY;	 /* if nonzero, drop out of the bootloader after this time */
 	bool try_boot;
+	_alert("Git commit: %s\n", GIT_COMMIT);
```

Comment:
How about just printing out "Version: " or such. We know it is git commit, but it doesn't need to say that, it can be more generic

Reply:
Changed to "Version: "